.github/CODEOWNERS

```diff
@@ -2,4 +2,4 @@
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
 
 *                     @iche033
-tutorials/*           @maryaB-osr
+*                     @WilliamLewww
```

Comment:
Ohh this pattern doesn't work, we always seem to do it though 😄 

* https://github.com/ignitionrobotics/ign-cmake/pull/155
* https://github.com/ignitionrobotics/ign-gui/pull/251